app/Services/User/RegisterUserService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Services\User;
+
+use App\Http\Requests\Api\Auth\RegisterRequest;
+use App\Http\Resources\ErrorResource;
+use App\Http\Resources\SuccessResource;
+use App\Http\Resources\UserResource;
+use App\Repositories\UserRepository;
+use Exception;
+use Illuminate\Http\Response;
+
+class RegisterUserService
+{
+    protected $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    public function execute(RegisterRequest $request)
+    {
+        try {
+            $data = $request->validated();
+
+            if( $request->hasFile('image')) {
```

Comment:
we don't have any attributes with name 'image' we have id_img and profile_img. and you should upload both.
Did you ensure that the image already uploaded and exists in the public folder..?